content/en/docs/release_notes/2_0_0_20210122_relnotes.md

```diff
@@ -1,7 +1,6 @@
 ---
 title: Axway CLI 2.0.0 GA release notes - 22 January 2021
 linkTitle: Axway CLI 2.0.0 GA release notes - 22 January 2021
-description: Review Axway CLI release details such as new features, improvements, breaking changes, and bug fixes.
```

Comment:
The front matter description is required to adhere to standards. Can you please add them back? I suggest using the same description for all Release Notes, instead of modifying them for each release. That way, when you create a new one, just use the same description and you don't have to worry about making it fit to the release. (The description is supposed to appear in a larger font in Zoomin to show it's header content, but something changed and it currently does not; the font size is the same as the body text and it looks odd. I have asked Paul to look into this and get it fixed with Zoomin.)

Reply:
Done, take a peek.